1/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM centos:centos7
+FROM openshift/origin:latest 
```

Comment:
we're deprecating the v1 image this release, i'd rather not touch it at this point.